COPYRIGHT.txt

```diff
@@ -0,0 +1,13 @@
+# Copyright Splunk Inc.
+#
+# Licensed under the Apache License, Version 2.0 \(the "License"\);
```

Comment:
The backslashes are for `pylintfileheader`?

Reply:
Yes. This whole file exists only for pylintfileheader. May be I should prefix it with a dot or suffix with `.tmpl`? Or just rename to `pylintheader_tmpl.txt` ?

Reply:
Nah, was just wondering :slightly_smiling_face: